lib/ansible/modules/packaging/os/apt.py

```diff
@@ -41,7 +41,7 @@
   cache_valid_time:
     description:
       - Update the apt cache if its older than the I(cache_valid_time). This option is set in seconds.
-        As of Ansible 2.4, this implicitly sets I(update_cache) if set.
+        As of Ansible 2.4, this implicitly sets I(update_cache) if not set.
```

Comment:
I think that the `if set` is tied to `I(cache_valid_time)` so the current version is correct but I am not a native speaker. 

cc @gundalow

Reply:
@rigarash how about this:
```
As of Ansible 2.4, this sets I(update_cache=yes)
```

Reply:
@mkrizek I think your version is far better than my proposal.

Reply:
@rigarash do you want to amend your PR with that then so we can merge it?

Reply:
@mkrizek Yes. Is adding the modifying commit to this PR is enough for the project? If so, I will prepare it.

Reply:
Yes, that's fine.

Reply:
@rigarash still interested in this?